tools/src/main/java/org/apache/kafka/tools/MetadataQuorumCommand.java

```diff
@@ -155,17 +169,35 @@ private static void handleDescribeReplication(Admin admin) throws ExecutionExcep
         );
     }
 
-    private static List<List<String>> quorumInfoToRows(QuorumInfo.ReplicaState leader, Stream<QuorumInfo.ReplicaState> infos, String status) {
-        return infos.map(info ->
-            Stream.of(
+    private static List<List<String>> quorumInfoToRows(QuorumInfo.ReplicaState leader,
+                                                       Stream<QuorumInfo.ReplicaState> infos,
+                                                       String status,
+                                                       boolean humanReadable) {
+        return infos.map(info -> {
+            String lastFetchTimestamp = !info.lastFetchTimestamp().isPresent() ? "-1" :
+                humanReadable ? format("%d ms ago", durationMs(info.lastFetchTimestamp().getAsLong())) :
+                    valueOf(info.lastFetchTimestamp().getAsLong());
+            String lastCaughtUpTimestamp = !info.lastCaughtUpTimestamp().isPresent() ? "-1" :
+                humanReadable ? format("%d ms ago", durationMs(info.lastCaughtUpTimestamp().getAsLong())) :
+                    valueOf(info.lastCaughtUpTimestamp().getAsLong());
+            return Stream.of(
                 info.replicaId(),
                 info.logEndOffset(),
                 leader.logEndOffset() - info.logEndOffset(),
-                info.lastFetchTimestamp().orElse(-1),
-                info.lastCaughtUpTimestamp().orElse(-1),
+                lastFetchTimestamp,
+                lastCaughtUpTimestamp,
                 status
-            ).map(r -> r.toString()).collect(Collectors.toList())
-        ).collect(Collectors.toList());
+            ).map(r -> r.toString()).collect(Collectors.toList());
+        }).collect(Collectors.toList());
+    }
+
+    private static long durationMs(long timestampMs) {
```

Comment:
Could we add a test for this method? It looks like we only test the output format, not test the output duration is correct.